packages/discovery-provider/ddl/migrations/0073_user_usdc_payout_address_history.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS user_payout_wallet_history (
+    user_id INTEGER NOT NULL,
+    spl_usdc_payout_wallet VARCHAR NOT NULL,
+    blocknumber INTEGER NOT NULL,
+    block_timestamp TIMESTAMP WITHOUT TIME ZONE NOT NULL,
```

Comment:
necessary?

Reply:
yeah, its what's used to determine the relevant payout wallet

Reply:
why do we need timestamps in addition to block numbers?

Reply:
solana has no concept of what acdc blocknumber correlates with what slot. The query looks for the most recent block timestamp of the payout wallets that's older than the Solana block timestamp when validating purchases

Reply:
check